### ISSUE TITLE
[tf2] -vulkan command line crashes the game with error message

### ISSUE THREAD

**Issue by flascaler** [ValveSoftware/Source-1-Games#5922]
everytime i try to use the -vulkan command line, it starts the game stuck on a black screen, eventually it'll say that it's not responding and an error message will popup
![image](https://github.com/ValveSoftware/Source-1-Games/assets/168214891/6f342418-f3d6-40d7-b409-fc27d0b66637)
i'm running windows 11 enterprise 64-bit, 8gb ram, intel i3-7020u with intel hd graphics 620 (driver version 31.0.101.2111, dx12 & vulkan 1.3.125).

**Comment by CaptainSandvich** [comment]
idk how this happened

**Comment by NotHyper-474** [comment]
Seems related to https://github.com/doitsujin/dxvk/issues/3575

**Comment by flascaler** [comment]
> Seems related to [doitsujin/dxvk#3575](https://github.com/doitsujin/dxvk/issues/3575)

do half-life 2, episodes & source not use the same version of dxvk as tf2? when i use the -vulkan or -gamepadui command line on them, they work perfectly fine. even fullscreen works, unlike dxvk for portal 2 & left 4 dead 2.

**Comment by NotHyper-474** [comment]
> > Seems related to [doitsujin/dxvk#3575](https://github.com/doitsujin/dxvk/issues/3575)
> 
> do half-life 2, episodes & source not use the same version of dxvk as tf2? when i use the -vulkan or -gamepadui command line on them, they work perfectly fine. even fullscreen works, unlike dxvk for portal 2 & left 4 dead 2.

I don't know about HL2 and etc. & Portal 2 but I know for sure L4D2 uses dxvk-native, and since you mentioned Portal also has the fullscreen issue they probably use the same version. 

But knowing which dxvk they use is very simple, dxvk always creates a (game name)_d3d9.log file in the executable's folder that generally shows which version is being used, you could try looking for that file.

**Comment by flascaler** [comment]
> > > Seems related to [doitsujin/dxvk#3575](https://github.com/doitsujin/dxvk/issues/3575)
> > 
> > 
> > do half-life 2, episodes & source not use the same version of dxvk as tf2? when i use the -vulkan or -gamepadui command line on them, they work perfectly fine. even fullscreen works, unlike dxvk for portal 2 & left 4 dead 2.
> 
> I don't know about HL2 and etc. & Portal 2 but I know for sure L4D2 uses dxvk-native, and since you mentioned Portal also has the fullscreen issue they probably use the same version.
> 
> But knowing which dxvk they use is very simple, dxvk always creates a (game name)_d3d9.log file in the executable's folder that generally shows which version is being used, you could try looking for that file.

it seems that they do use different dxvk versions. hl2 uses v1.8.1-3-gd3e43ed3 and tf2 uses v2.3-34-g1568c263

**Comment by flascaler** [comment]
searching on the intel website, there seems to be a graphics driver version, 31.0.101.2127, which is listed to support intel i3-7020u, the cpu i have that also contains the intel hd 620 igpu. it seems that it just doesn't install automatically from windows updates or intel driver and support assistant. i'll attempt to install it and try again.

**Comment by flascaler** [comment]
i installed the new driver, and now it crashes without any error message.
before this, i forgot to delete the dxvk log file that was from the older driver version, which gave me a slightly altered error message than the initial one.
![j](https://github.com/ValveSoftware/Source-1-Games/assets/168214891/3fb2d191-bfcb-4921-a10e-c7275d4d535e)

**Comment by NotHyper-474** [comment]
Yeah, seems like unfortunately that GPU's not gonna work for DXVK 2.0+. I'd suggest replacing the DXVK dll with an older version so at least you'd able to use Vulkan but I think that would risk you getting a VAC ban.

I don't think it would make any difference anyway, using Vulkan doesn't necessarily mean you're gonna get magically increased performance (which is what I assume you're using Vulkan for in the first place)

**Comment by flascal** [comment]
update on this, i'm still using the same laptop but now on windows 10 iot ltsc 21h2. intel drivers 31.0.101.2134, vulkan runtime 1.4.309
i managed to get the game into a turbine match on casual with no issues until it crashed once more (the crash message is the same as the two previous error messages, just in french and the memory addresses are different)

![Image](https://github.com/user-attachments/assets/0f36eecf-9f8c-4133-9542-772a5858fefa)

i'll probably stop updating this now because i'm planning to get a new computer soon so this won't really matter to me anymore
